File: java/src/main/java/org/npb/agview/api/example/MovementAddresses.java
package org.npb.agview.api.example;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class MovementAddresses {
    private int movementId;
    private String source;
    private String sourceLatitude;
    private String sourceLongitude;
    private String sourceStreetAddress;
    private String sourceCity;
    private String sourceState;
    private String sourceZip;

    private String target;
    private String targetLatitude;
    private String targetLongitude;
    private String targetStreetAddress;
    private String targetCity;
    private String targetState;
    private String targetZip;
}

File: c-sharp/Api/MovementAddresses.cs
using CsvHelper.Configuration.Attributes;

namespace Npb.Agview.Api.Example
{
    public class MovementAddresses
    {
        [Name("movement_id")]
        public int MovementId {get; set;}


        [Name("source")]
        public string Source {get; set;}

        [Name("source_latitude")]
        public string SourceLatitude {get; set;}

        [Name("source_longitude")]
        public string SourceLongitude {get; set;}

        [Name("source_street_address")]
        public string SourceStreetAddress {get; set;}

        [Name("source_city")]
        public string SourceCity {get; set;}

        [Name("source_state")]
        public string SourceState {get; set;}

        [Name("source_zip")]
        public string SourceZip {get; set;}


        [Name("destination")]
        public string Target {get; set;}

        [Name("destination_latitude")]
        public string TargetLatitude {get; set;}

        [Name("destination_longitude")]
        public string TargetLongitude {get; set;}

        [Name("destination_street_address")]
        public string TargetStreetAddress {get; set;}

        [Name("destination_city")]
        public string TargetCity {get; set;}

        [Name("destination_state")]
        public string TargetState {get; set;}

        [Name("destination_zip")]
        public string TargetZip {get; set;}
    }
}